src/pages/index.tsx

```diff
@@ -1,75 +1,88 @@
-        }
-        .header {
-          font-size: 72px;
-          background: -webkit-linear-gradient(#eee, ${theme.palette.accents_2});
-          -webkit-background-clip: text;
-          -webkit-text-fill-color: transparent;
-        }
+    const theme = useTheme();
+    const [useMobileBar, setMobileBar] = useState(false);
+    dynamicWidth((width) => {
+        width < 1200 ? setMobileBar(true) : setMobileBar(false);
+    });
+    return (
+        <>
+            <div className={"video__player"}>
+                <video autoPlay={true} muted loop style={{height: "100%", width: "100%"}}>
+                    <source type="video/webm" src="/assets/marketing/header_animated.webm" height="100%" width="100%"/>
+                </video>
+            </div>
+            <div style={{position: "relative", zIndex: "1"}}>
+                <div className="text-center">
+                    <Text h1 className="header" style={{fontSize: "calc(2rem + 2vw)"}}>
+                        Your PC but even better.
+                    </Text>
+                </div>
+                <Alert/>
+                <Spacer h={1}/>
+                <div className="center card-container">
+                    <div
+                        style={{display: 'flex', flexDirection: 'row', justifyContent: 'center', alignItems: 'center'}}>
+                        {useMobileBar ? null : (
+                            <NextLink href="/demo">
+                                <Button shadow type="secondary" margin="10px">
+                                    Try in your browser
+                                </Button>
+                            </NextLink>
+                        )}
+                        <NextLink href="/downloads">
+                            <Button shadow type="success" margin="10px">
+                                Download Now
+                            </Button>
+                        </NextLink>
+                    </div>
+                </div>
+                <Spacer h={4}/>
+                <Discover/>
+                <Spacer h={2}/>
+            </div>
+            <style jsx>{`
+                .card-container {
+                    display: flex;
+                    flex-direction: column;
+                    justify-content: center;
+                    align-items: center;
+                    margin: 0 auto;
+                    max-width: 500%;
+                    padding: 0;
+                    border-top: none !important;
+                }
+                .header {
+                  background: -webkit-linear-gradient(#eee, ${theme.palette.accents_2});
+                  -webkit-background-clip: text;
+                  -webkit-text-fill-color: transparent;
+                }
+      
+                .video__player {
+                    width: 100vw;
+                    position:relative;
+                    margin-bottom: ${useMobileBar ? "-20vh" : "-60vh"};
+                }
+                
```

Comment:
it's okay to group related style definition with spaces, but this is unnecessary spacing. You might want to review your css definition and properly group definitions.